### DISCUSSION TITLE
只狼 影逝二度|豪华中文|最终年度版-影月剑魂+预购特典+全DLC+修改器-支持手柄|解压即撸

### DISCUSSION THREAD

**Discussion by match-fire** [share121/inter-knot#1431] [其他]
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/814380/ss_2685dd844a2a523b6c7ec207d46a538db6a908cd.1920x1080.jpg?t=1603904569)
在《Sekiro: Shadows Die Twice》中你是“独臂之狼”，一个名誉不再、伤痕累累的忍者，

一个从死亡边缘捡回一命的战士。你效忠守护继承古老血统的年轻皇子，与危险的苇名一族以及众多凶恶之徒为敌。

年轻的皇子被抓走后，为挽回荣誉，你将不畏死亡，踏上危机四伏的征程。

探索生死冲突不断的16世纪后期，感受残酷的日本战国时代，在黑暗、扭曲的世界，与威胁生命的敌人对峙。

活用义手装备各种致命武器，大显忍者身手，在血腥对抗中潜行、上下穿梭，与敌人正面激烈交锋。

复仇雪耻。夺回荣誉。巧妙杀敌。
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/814380/ss_53d73aff4d439cbd9b9b69430f9b3b6db8d0ecd7.1920x1080.jpg?t=1603904569)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/814380/ss_2685dd844a2a523b6c7ec207d46a538db6a908cd.1920x1080.jpg?t=1603904569)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/814380/ss_15f0e9982621aed44900215ad283811af0779b1d.1920x1080.jpg?t=1603904569)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/814380/ss_1e6f5540866a5564d65df915c22fe1e57e336a6f.1920x1080.jpg?t=1603904569)
![](https://shared.st.dl.eccdnx.com/store_item_assets/steam/apps/814380/ss_3d6b38c382c0eafb02dc90d22f33fd292e4e5cf3.1920x1080.jpg?t=1603904569)

链接: https://pan.baidu.com/s/1q7vzc6B8DDzAjkAobcibrg

如遇到有带x的提取码请手打输入: itbh

解压密码:laoquzhang.com

**Comment by zzzuser46** [comment]
绳网还真是什么类型的帖子都有，话说在新艾丽都怎么玩只狼

**Comment by 13579yangh** [comment]
可以超弦一郎吗

**Comment by lllliufeng** [comment]
我去、老司机

**Comment by zifeiyu1111** [comment]
【委托】雾隐贵人怎么打？
